tests/Testcases/IPGenerate/IP_Catalog/dummy_generators/RapidSilicon/IP/axis_converter/V1_0/axis_converter_gen.py

```diff
@@ -1,17 +1,21 @@
 #!/usr/bin/env python3
```

Comment:
How are we invoking the this script in linux ? This shebang will only work there.

Reply:
Just a copy paste from [IP_Catalog](https://github.com/RapidSilicon/IP_Catalog/blob/202508e79ef2d0fc37a8513f9d206d1e6322d019/RapidSilicon/IP/axi_ram/v1_0/axi_ram_gen.py#L1) I assume this only allows unix to execute the script w/o specifying the python path, but i think in all our uses we execute the script like "<path to python3> script_gen.py" so I don't think the shebang matters.

Reply:
Got it, this isn't resolved yet.
The IP Catalog needs to adopt these changes too, it got dropped off my radar last time I saw it.
Please move this issue to pre-jira backlog internally.
https://docs.python.org/3/using/windows.html#python-launcher-for-windows.
I will mark this as resolved now.